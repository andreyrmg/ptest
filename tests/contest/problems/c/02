1
abc def ghi jkl mno prs tuv xyz.#