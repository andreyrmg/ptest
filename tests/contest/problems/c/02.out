22#2223133#3334144#4445155#5556166#66671778#8818889#9910#00211